src/librustc/traits/project.rs

```diff
@@ -206,15 +206,15 @@ pub fn poly_project_and_unify_type<'cx, 'gcx, 'tcx>(
 
     let infcx = selcx.infcx();
     infcx.commit_if_ok(|snapshot| {
-        let (skol_predicate, skol_map) =
-            infcx.skolemize_late_bound_regions(&obligation.predicate);
+        let (skol_predicate, placeholder_map) =
```

Comment:
Maybe find a better name for `skol_predicate`?
(multiple occurences in other files)